graylog2-web-interface/src/pages/SearchPage.jsx

```diff
@@ -30,6 +30,7 @@ const SearchPage = React.createClass({
     Reflux.connect(CurrentUserStore),
     Reflux.listenTo(InputsStore, '_formatInputs'),
     Reflux.listenTo(RefreshStore, '_setupTimer', '_setupTimer'),
+    Reflux.listenTo(DecoratorsStore, '_refreshDataFromDecoratorStore', '_refreshDataFromDecoratorStore')
```

Comment:
Why are we using strings here instead of a function handle?

```javascript
Reflux.listenTo(DecoratorsStore, this._refreshDataFromDecoratorStore, this._refreshDataFromDecoratorStore);
```